src/aims_ui/page_postcode.py

```diff
@@ -38,6 +38,9 @@ def postcode():
       all_user_input,
   )
 
+  if result == 'error_connecting':
+    return page_error(None, page_name, connection_error=True)
```

Comment:
Likewise, this looks like more error-checking code that could just be a `catch`.